x/gamm/keeper/math.go

```diff
@@ -98,6 +98,22 @@ func calcOutGivenIn(
 	return tokenAmountOut
 }
 
+// exported version
```

Comment:
nit: let's add proper, albeit it can be concise, godoc on what this does :)

```suggestion
// CalcOutGivenIn ...
```